### PR TITLE
feature/console-tests

### PR DESCRIPTION
**Pull request by Ivan-Marquez**
## Description
Includes unit tests for the **console** package in order to increase code coverage.

## Related issues
- #57 

## Achievements
- Increased package code coverage by 53%.
- Reduced linter warnings by adding comments for exported functions.

## Notes
Hello @sqshq ,

This is what I got so far regarding tests for the **console** package. I tried my best to avoid brittle tests and just focused on the exported functions. I also started to document exported functions in order to reduce warnings generated by `go-lint`. 

Any feedback would be greatly appreciated. Thank you @sqshq !

### CODE REVIEW

File: console/palette.go [hunk 1]
```diff
@@ -2,33 +2,52 @@ package console
 
 import (
 	"fmt"
-	ui "github.com/gizak/termui/v3"
 	"runtime"
+
+	ui "github.com/gizak/termui/v3"
 )
 
+// Theme is a representation of a theme
```
Comment (sqshq) [inline console/palette.go]: Hi @Ivan-Marquez,
Thank you for the contribution. Regarding changes in this file - I know that linter is complaining about lack of comments here, but I'm not a big fun of uninformative comments like `type Theme: Theme is a representation of a theme`

The code is clear without such comments, so I would prefer leave the constants without them.

Comment (Ivan-Marquez) [reply console/palette.go]: Hello @sqshq ,

Thank you so much for your feedback! I'll be updating this PR with the required changes.

Comment (Ivan-Marquez) [reply console/palette.go]: @sqshq PR updated with requested changes! And it totally makes sense. The names of the constants are already sufficiently descriptive.

Comment (sqshq) [reply console/palette.go]: thank you!

**Pull request merged**